### DISCUSSION TITLE
How to realize state management with riverpod?

### DISCUSSION THREAD

**Discussion by Zviryasha** [serverpod/serverpod#1539] [Q&A]
Hi I am new in programing
On youtube i found this [tutorial](https://www.youtube.com/watch?v=oSwT9FZ18j4&list=PLgRx2Eap1Wm1pH9fZz3ISQoCJ-I6wmxFS&ab_channel=TylerCodes)
In this tutorial Tyler uses StateNotifier and StateNotifierProvider.
But can anyone help me how to do error handling and loading?
[Source code](https://github.com/tylersavery/full-stack-dart)

**Comment by SandPod** [comment]
Hey @Zviryasha 👋 

This questions seems to be more concerned around the use the Riverpod framework.
Unless there is a specific question around the Serverpod framework I would suggest you bring the question up in their discussions forum: https://github.com/rrousselGit/riverpod/discussions
